core-bundle/src/Filesystem/UnableToResolveUuid.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Filesystem;
+
+use League\Flysystem\FilesystemException;
+use Symfony\Component\Uid\Uuid;
+
+class UnableToResolveUuid extends \RuntimeException implements FilesystemException
```

Comment:
Afaik we use the `Exception` suffix everywhere.

Reply:
I know, I was torn between our naming convention and placing the class in the `Exception` namespace versus following the flysystem convention https://github.com/thephpleague/flysystem/tree/2.x/src. Still not sure :stuck_out_tongue:.

Reply:
Imho it should be consistent across our bundle, not the PHP universe :)

Reply:
see 0b70134f9d7456b3c8862ac203ce120dadce8405